traffic-policy/actions/forward-internal/behavior.mdx

```diff
@@ -0,0 +1,7 @@
+## Behavior
+
+### On Error
```

Comment:
```suggestion
### Managing Fallback Behavior (`on_error`)
```

Maybe we add Managing / Enabling Fallback here so people looking for that capability can easily find it without having to read through the paragraphs.

Reply:
I like it, I spent too long trying to figure out a good heading for this and ended up with the most basic one lol